### ISSUE TITLE
How can I continue processing audio while returning it using MultimodalAgent/Openai realtime api

### ISSUE THREAD

**Issue by phpmaple** [livekit/agents#910]
For example, voice conversion

**Comment by theomonnom** [comment]
Hey, can you provide more details? Are you looking into getting the audio data from the Realtime API?

**Comment by phpmaple** [comment]
Yes, I want to do some audio processing before returning the audio stream to the user and then return the processed audio result to the user

**Comment by phpmaple** [comment]
#953 I have submitted a PR
